frontend/model/database.js

```diff
@@ -66,3 +68,24 @@ sbp('sbp/selectors/register', {
     return files.removeItem(url)
   }
 })
+
+// =======================
+// Save recent notifications here
+// =======================
+
+const notificationStorage = localforage.createInstance({
+  name: 'Group Income',
+  storeName: 'Notifications'
+})
+
+sbp('sbp/selectors/register', {
+  'gi.db/notifications/save': function (username: string, items: Notification[]): Promise<void> {
+    return notificationStorage.setItem(username, items)
+  },
+  'gi.db/notifications/load': function (username: string): Promise<Notification[]> {
+    return notificationStorage.getItem(username)
+  },
+  'gi.db/notifications/delete': function (username: string): Promise<void> {
+    return notificationStorage.removeItem(username)
+  }
+})
```

Comment:
Are these never used? If so, shouldn't they be deleted? Or is this PR unfinished?

Reply:
Good catch. I'm deleting this for now.